README.md

```diff
@@ -68,6 +68,11 @@ Update your local `hosts` file with the remote server's ip address
 remote-server-ip-address (maintain other default options)
 ```
 
+If the SSH key you intend to use isn't `~/.ssh/id_rsa.pub`, change the location in `group_vars/all`    
+```
+ssh_public_key_path: "~/.ssh/id_rsa.pub"
+```
+
```

Comment:
Since there are several ways to change this variable (the `hosts` file, environment variables, and `group_vars/all`), I'm wondering whether we should be a bit less concrete here :thinking:. How about something like:

```suggestion
If the SSH key you intend to use isn't `~/.ssh/id_rsa.pub`, change the location using the `ssh_key_path` [configuration variable](#configuration-variables).
```

Maybe we could also expand the "Configuration Variables" section as well, maybe linking to the Ansible documentation :thinking:. Just thinking aloud here; I'm not saying you should add it to this pull request :wink:.

Reply:
I did try to change it in `hosts` and only there at first and running the playbook that way would result in an error, because the process was not using my updated SSH key location. It's only after I changed the location in `group_vars/all` that the process actually used the new location. 

Now, I'm not very familiar with Ansible and I am not sure why this happens. But it seems to me like just changing the var in `hosts` is not enough.

Reply:
I think you mentioned that you changed the `ansible_ssh_private_key_file` variable in the `hosts` file, right? :thinking: The `ansible_ssh_private_key_file` and `ssh_public_key_path` are different; one is for the private key to use when connecting and the other one for a public key you'd like to install on the server (which is useful if you're using a password to connect to the server via SSH).

Indeed, very confusing :thinking:.

How about putting those two variables together, so it's more obvious that they're different? What do you think about this patch?

```diff
diff --git a/README.md b/README.md
index ddaf6bc..0134b9d 100644
--- a/README.md
+++ b/README.md
@@ -235,7 +235,8 @@ timezone: Europe/Madrid
 locale: en_US.UTF-8
 
 # Authorized Hosts
-ssh_public_key_path: "change_me/.ssh/id_rsa.pub"
+ssh_public_key_path: "~/.ssh/id_rsa.pub"
+ansible_ssh_private_key_file: "~/.ssh/id_rsa"
 
 #Postgresql
 database_name: "consul_production"
diff --git a/group_vars/all b/group_vars/all
index 67948b7..f0295fb 100644
--- a/group_vars/all
+++ b/group_vars/all
@@ -25,6 +25,7 @@ shared_public_dirs:
   - "ckeditor_assets"
 
 ssh_public_key_path: "~/.ssh/id_rsa.pub"
+ansible_ssh_private_key_file: "~/.ssh/id_rsa"
 
 #Postgresql
 database_name: "{{app_name}}_{{ env }}"
diff --git a/hosts.example b/hosts.example
index 98d7a2d..dd1384e 100644
--- a/hosts.example
+++ b/hosts.example
@@ -1,2 +1,2 @@
 [servers]
-remote-server-ip-address ansible_user=root ansible_ssh_extra_args='-o StrictHostKeyChecking=no' ansible_ssh_private_key_file='~/.ssh/id_rsa' ansible_port=22
+remote-server-ip-address ansible_user=root ansible_ssh_extra_args='-o StrictHostKeyChecking=no' ansible_port=22
```

Could you try the installer with this patch and then change the variables you need in the `hosts` file?

Reply:
Yes, it worked this way! 

Sadly, I wasn't able to apply your git patch, I just modified the files myself. 

Shall I go ahead and push the modified files to this PR, then get the PR up to date with the master branch and then maybe we can merge it?

Reply:
> Shall I go ahead and push the modified files to this PR, then get the PR up to date with the master branch and then maybe we can merge it?

Yes, that would be awesome :tada:. Thanks! :pray: